src/components/Content.js

```diff
@@ -54,16 +54,27 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => ({
   showRemindMeLaterModal: () => dispatch({ type: 'modalRemindMeLater', MODAL_CLOSE_DURATION }),
-  cursorBack: () => dispatch(cursorBack())
+  cursorBack: () => dispatch(cursorBack()),
+  toggleSidebar: () => dispatch({ type: 'toggleSidebar' })
 })
 
-const Content = props => {
-  const { search, isTutorial, tutorialStep, showModal, showRemindMeLaterModal, cursorBack: moveCursorBack, rootThoughts, noteFocus } = props
+const isLeftSpaceClick = (event, content) => {
```

Comment:
Please add a JSDOC comment with at minimum a description of the function and its return value to all function definitions. Exceptions are conventional React/Redux functions (like `mapStateToProps`) and extremely trivial functions.